diff --git a/src/components/TextInput/Adornment/TextInputIcon.tsx b/src/components/TextInput/Adornment/TextInputIcon.tsx
index 3b1a84ff9cf12a0f325d78c4721b6508049259d9..bd38ca64abb924294b79f501b170b095e9c4a917 100644
--- a/src/components/TextInput/Adornment/TextInputIcon.tsx
+++ b/src/components/TextInput/Adornment/TextInputIcon.tsx
@@ -129,7 +129,7 @@ const IconAdornment: React.FunctionComponent<
 const TextInputIcon = ({
   icon,
   onPress,
-  forceTextInputFocus,
+  forceTextInputFocus = true,
   color: customColor,
   theme: themeOverrides,
   rippleColor,
@@ -176,10 +176,6 @@ const TextInputIcon = ({
 };
 TextInputIcon.displayName = 'TextInput.Icon';
 
-TextInputIcon.defaultProps = {
-  forceTextInputFocus: true,
-};
-
 const styles = StyleSheet.create({
   container: {
     position: 'absolute',
